docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -364,8 +365,30 @@ EXPOSE 8080
 CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
 ----
 
+Sample Dockerfile for building with Gradle:
+[source,dockerfile,subs=attributes+]
+----
+## Stage 1 : build with maven builder image with native capabilities
+FROM quay.io/quarkus/centos-quarkus-maven:{graalvm-version}-java8 AS build
+COPY src /usr/src/app/src
+COPY build.gradle /usr/src/app
+COPY settings.gradle /usr/src/app
+COPY gradle.properties /usr/src/app
+USER root
+RUN chown -R quarkus /usr/src/app
+USER quarkus
+RUN gradle -b /usr/src/app/build.gradle clean buildNative
+
+## Stage 2 : create the docker final image
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+WORKDIR /work/
+COPY --from=build /usr/src/app/build/*-runner /work/application
+RUN chmod 775 /work
+EXPOSE 8080
+CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
+----
 
-Save this file in `src/main/docker/Dockerfile.multistage` as it is not included in the getting started quickstart.
+Pick one of the above Dockerfile samples and save it in `src/main/docker/Dockerfile.multistage` as it is not included in the getting started quickstart.
```

Comment:
This needs a bit of rewording IMHO because the getting started project is a Maven one. So it makes no sense putting the Gradle one there.

I would put that sentence just after the Maven example. And then have something like "If you are using Gradle in your project, you can use this sample Dockerfile:"